gems/aws-sigv4/spec/aws-sigv4/signer_spec.rb

```diff
@@ -5,6 +5,24 @@
 require 'tempfile'
 require 'base64'
 
```

Comment:
Isn't this defined in spec helper already? Also I think you just need to use defined?

Reply:
It was but wasn't working correctly, I've fixed it.